content/_help/specific-agencies/ssa._en.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: help
+title: SSA.gov
+category: specific-agencies
+permalink: /help/specific-agencies/ssa/
+keywords: social security, social security benefits, replacement social security card, social security card, ssa, social security card replacement, social security statement, replace social security card, new social security card, new card, replacement card, ss statement, social security account, social security administration, ssa.gov, i need a new social security card
+order: 3
+redirect_from:
+---
+Login.gov can only answer questions about the sign-in process and creating a Login.gov account. Your Login.gov account does not affect or have any information about your [Social Security](https://www.ssa.gov/) card, number, or benefits. 
+
+Please do not send Login.gov sensitive data about yourself or identifying numbers.
+
+Login.gov can only answer questions about the sign-in process and creating a Login.gov account.
+
+If you already have an [SSA.gov](https://www.ssa.gov/) account, you do not need to create a Login.gov account unless required by SSA.
+
+Please contact [SSA.gov](https://www.ssa.gov/) directly if you have questions regarding:
+
+* [Your Social Security number](https://www.ssa.gov/employer/ssnv.htm)
+* [Replacing your Social Security card](https://www.ssa.gov/number-card/replace-card)
+* [Reporting a stolen Social Security number](https://www.ssa.gov/number-card/report-stolen-number)
+* Other benefits questions (SSA FAQ page: [https://faq.ssa.gov/en-US/](https://faq.ssa.gov/en-US/))
+* Checking eligibility for:
+  * [Retirement](https://www.ssa.gov/retirement)
+  * [Disability Benefits](https://www.ssa.gov/benefits/disability/)
+  * [Supplemental Security Income (SSI)](https://www.ssa.gov/ssi)
+  * [Medicare](https://www.ssa.gov/medicare)
+
+Contact the 1-800-772-1213 between 8:00 a.m. – 7:00 p.m. Monday through Friday. TTY number at 1-800-325-0778. or submit your request via web form at [https://secure.ssa.gov/emailus/](https://secure.ssa.gov/emailus/).
```

Comment:
Can we double-check with the UX folks on the phrasing here? It feels like it's missing some words after "Contact the", or maybe we just want to drop the "the".

Reply:
See discussion here for some revisions to the text, as well as the link in the paragraph below: https://docs.google.com/document/d/1cBGm3kjf8iRVJ7REEjpuSsJj9F0onPHNzoCbaPQ_Sas/edit?disco=AAAA-lTg1Aw